src/utils/constants.ts

```diff
@@ -2,3 +2,7 @@
 // @ts-ignore
 export const MAX_LOCK_ROUNDS: i64 = 26;
 export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+export const GARDEN_POOL = 'gardenPool';
+export const GIVBACK = 'givback';
+export const GIVDROP = 'givdrop';
+export const UNISWAP_POOL = 'uniswapPool';
```

Comment:
Why uniswap pool? just `UNIPOOL` is correct. Because every farm will have this value.